src/Classes/Utils/Presets.ts

```diff
@@ -0,0 +1,208 @@
+import { RecursivePartial } from "../../Types/RecursivePartial";
+import { IOptions } from "../../Interfaces/Options/IOptions";
+import { MoveDirection } from "../../Enums/MoveDirection";
+import { OutMode } from "../../Enums/OutMode";
+import { PresetType } from "../../Enums/PresetType";
+import { ShapeType } from "../../Enums/ShapeType";
+import { HoverMode } from "../../Enums/Modes/HoverMode";
+
+export class Presets {
+    public static getPreset(key: PresetType): RecursivePartial<IOptions> | undefined {
+        return this.available[key.toString()];
+    }
+
+    private static get available(): { [key: string]: RecursivePartial<IOptions> } {
+        return {
+            [PresetType.basic]: this.basic,
+            [PresetType.backgroundMask]: this.backgroundMask,
+            [PresetType.fontAwesome]: this.fontAwesome,
+            [PresetType.snow]: this.snow,
+            [PresetType.bouncing]: this.bouncing,
+            [PresetType.stars]: this.stars,
+        }
+    }
+
+    private static get basic(): RecursivePartial<IOptions> {
+        return {
+            particles: {
+                color: {
+                    value: "#ffffff",
+                },
+                lineLinked: {
+                    color: "#ffffff",
+                    distance: 150,
+                    enable: true,
+                    opacity: 0.4,
+                    width: 1,
+                },
+                move: {
+                    direction: MoveDirection.none,
+                    enable: true,
+                    outMode: OutMode.out,
+                    speed: 2,
+                },
+            },
+        };
+    }
+
+    private static get backgroundMask(): RecursivePartial<IOptions> {
+        return {
+            particles: {
+                color: {
+                    value: "#ffffff",
+                },
+                lineLinked: {
+                    color: "#ffffff",
+                    distance: 150,
+                    enable: true,
+                    opacity: 1,
+                    width: 1,
+                },
+                move: {
+                    direction: MoveDirection.none,
+                    enable: true,
+                    outMode: OutMode.out,
+                    random: false,
+                    speed: 2,
+                    straight: false,
+                },
+                opacity: {
+                    random: false,
+                    value: 1,
+                },
+                shape: {
+                    type: ShapeType.circle,
+                },
+                size: {
+                    random: true,
+                    value: 30,
+                },
+            },
+            interactivity: {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The key 'interactivity' is not sorted alphabetically](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5216054)